tests/lib/eslint/eslint.js

```diff
@@ -1525,26 +1526,28 @@ describe("ESLint", () => {
 
                 it("should throw if eslint.config.js file is not present even if overrideConfig was passed", async () => {
                     eslint = new ESLint({
+                        flags,
                         cwd: getFixturePath(".."),
                         overrideConfig: {
                             rules: {
                                 "no-unused-vars": 2
                             }
                         }
                     });
-                    await assert.rejects(() => eslint.lintFiles("fixtures/undef*.js"), /Could not find config file/u);
```

Comment:
There's a config file in "fixtures", so this test wouldn't work with the `unstable_config_lookup_from_file` flag.